pp_sort.c

```diff
@@ -179,7 +175,7 @@ typedef SV * gptr;              /* pointers in our lists */
 */
 
 
-static IV
+PERL_STATIC_FORCE_INLINE IV __attribute__always_inline__
```

Comment:
I think here and another place, the __attribute__always_inline__ is a relic of an earlier version, and is generated by the macro.  Most functions declared as such in this file don't have an __attribute__ specified explicitly

Reply:
I've added `sortsv_flags_impl` to `embed.fnc` and removed the attribute from its definition but I would rather not do the same for `dynprep()`.

`dynprep()` takes arguments of types that exist only inside `pp_sort.c`, so I'd have to either move the typedefs to `perl.h` (which is an obfuscation) or remove them (which is beyond the scope of this PR).